content/tutorials/channel-rewind.textile

```diff
@@ -0,0 +1,217 @@
+      var URL = `https://realtime.ably.io/sse?v=1.1&key=${apiKey}&channels=${channelWeather}`
+      eventSource = new EventSource(URL);
+      eventSource.onmessage = function(msg) {
+          var message = JSON.parse(msg.data);
+          var weatherData = JSON.parse(message.data);
+          $resultWeather.text((weatherData.main.temp - 273.15).toFixed(2) + '°C with ' + weatherData.weather[0].description);
+      }
+      eventSource.onerror = function(error) {
+          if(error.data) {
+              $resultWeather.text(`Error: ${error.data}`);
+          }
+      }
+    }
+</script>
+```
+
+If you run this file in your browser now, you should see the realtime data coming through from the weather data stream via SSE. Notice that we haven't used any client SDKs here. Also, we get the latest message that was previously published on that channel, even if a new update isn't available at the time.
+
+h3(#using-mqtt). Option 3: Consume rewound and new data using Ably's MQTT protocol adapter
+
+You can use any MQTT library to publish and receive messages to and from Ably. For this tutorial, we'll make use of the "MQTT NPM library":https://www.npmjs.com/package/mqtt, but feel free to use any that you like.
+
+Create a new javascript file, say @main.js@. We'll begin by requiring the MQTT library and setting the correct options for our client to use Ably:
+
+```[javascript]
+var mqtt = require('mqtt');
+var options = {
+    keepalive: 30,
+    username: '<FIRST-HALF-OF-YOUR-ABLY-API-KEY>',
+    password: '<SECOND-HALF-OF-YOUR-ABLY-API-KEY>',
+    port: 8883
+};
+```
+
+Don't forget to replace the placeholders in the above snippet with your own API key. 
+
+Next, we need to connect to Ably's MQTT endpoint and subscribe to the channel that we are interested in, with the rewind parameter enabled of course:
+
+```[javascript]
+var client = mqtt.connect('mqtts:mqtt.ably.io', options);
+client.on('connect', () => {
+    //subscribe to the channel streaming the weather data for London, UK
+    client.subscribe('[product:ably-openweathermap/weather?rewind=2]weather:2643741');
+});
+```
+
+When a message comes through as a callback to this subscribe method, we'll simply log it to the console:
+
+```[javascript]
+client.on('message', (topic, message) => {
+    var msg = JSON.parse(message);
+    console.log((msg.main.temp - 273.15).toFixed(2) + '°C with ' + msg.weather[0].description)
+});
+
+client.on('error', (topic, message) => {
+    console.log(message.toString())
+});
+```
+
+h2(#live-demo). Live demo
+
+<iframe src="https://fourth-estimate.glitch.me/" height="200px" width="100%" frameBorder="0" ></iframe>
+
+h2(#conclusion). Conclusion
+
+In this tutorial, we saw the usage of rewind channel parameter in a real-world data streaming application. We saw three different ways of its usage, i.e., using Ably's native client SDKs, Ably's SSE endpoint and Ably's MQTT endpoint. 
```

Comment:
`we saw the usage of rewind channel parameter` => `we saw the usage of the rewind channel parameter`

Reply:
`three different ways of its usage` => `three different ways to use it`